configs/registration_local.properties

```diff
@@ -5,6 +5,7 @@ registration.port=8085
 registration.registrationCredential.accountId=registration_service@notary
 registration.registrationCredential.pubkeyPath=deploy/iroha/keys/registration_service@notary.pub
 registration.registrationCredential.privkeyPath=deploy/iroha/keys/registration_service@notary.priv
+registration.notaryAccount=notary@notary
```

Comment:
I would name it as "registration .client_storage_account". Notary is not meaningful.